### PR TITLE
chore(images): upload images to bucket

### PR DESCRIPTION
**Pull request by MichaelSolati**

### CODE REVIEW

File: src/site/content/en/handbook/grammar/index.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ description: |
 ---
 
 <style>
-  img[src*="icon"] {
+  img.icon {
```
Comment (robdodson) [inline src/site/content/en/handbook/grammar/index.md:10]: can you get rid of the `img` part of this?

File: src/site/content/en/handbook/grammar/index.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ description: |
...
     max-width: 32px;
     vertical-align: middle;
   }
 
-  img[src*="bordered"] {
+  img.icon.bordered {
```
Comment (robdodson) [inline src/site/content/en/handbook/grammar/index.md:15]: same comment as above

File: src/site/content/en/handbook/self-assessment-components/self-assessment.assess.yml [hunk 3]
```diff
@@ -83,17 +83,17 @@ questions:
     stem: "Which **two** Sesame Street muppets are clearly the best?"
     columns: true
     options:
-      - content: "![Elmo](./elmo.jpg)"
+      - content: "![Elmo](image/tcFciHGuF3MxnTr1y5ue01OGLBn2/ng9hB0jbzGXy0MY7upMU.jpg)"
```
Comment (robdodson) [inline src/site/content/en/handbook/self-assessment-components/self-assessment.assess.yml:86]: does this get transformed into a full image path?

File: src/site/content/en/lighthouse-pwa/codelab-apple-touch-icon/index.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ the example app to your iOS device's home screen when an Apple touch icon hasn't
 1. Open Safari for iOS.
 1. Open the URL of your example app. This is the URL like `https://example.glitch.me` that you
    noted earlier.
-1. Tap **Share** <img style="height:1.2em;vertical-align:top;" src="share.png"/>  >
+1. Tap **Share** {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/79A8XsdBaHeLGY5Wdm1J.png", alt="Apple Share Button", width="15", height="20", style="vertical-align:top;" %}  >
```
Comment (robdodson) [inline src/site/content/en/lighthouse-pwa/codelab-apple-touch-icon/index.md:58]: It seems like they were purposefully trying to make this image larger than its default height. 1.2em would usually be 19.2px

File: src/site/content/en/live/index.njk [hunk 5]
```diff
@@ -30,16 +30,22 @@ pageScripts:
...
-                <img src="live.png" alt="live" />
-              </video>
+            {% Video
+              src="video/tcFciHGuF3MxnTr1y5ue01OGLBn2/vBhlsAXOy80qltOoAZ3J.mp4",
+              width="156",
+              height="72",
+              autoplay=true,
+              muted=true,
+              loop=true,
+              playsinline=true,
```
Comment (robdodson) [inline src/site/content/en/live/index.njk:40]: it seems like `playsinline` isn't a standalone attribute but is instead part of autoplay
https://github.com/GoogleChrome/webdev-infra/blob/main/shortcodes/Video.js#L81

**Review by robdodson — APPROVED** [review]
LGTM with comments

File: src/site/content/en/handbook/self-assessment-components/self-assessment.assess.yml (hunk 3 above)
Comment (MichaelSolati) [reply src/site/content/en/handbook/self-assessment-components/self-assessment.assess.yml:86]: Yes, the Assessment shortcode will go through all the img tags and convert them to use imgix.

File: src/site/content/en/live/index.njk (hunk 5 above)
Comment (MichaelSolati) [reply src/site/content/en/live/index.njk:40]: Updated the library to support it.

File: src/site/content/en/lighthouse-pwa/codelab-apple-touch-icon/index.md (hunk 4 above)
Comment (MichaelSolati) [reply src/site/content/en/lighthouse-pwa/codelab-apple-touch-icon/index.md:58]: fixed, i set the width and height in em in the style too so it doesnt look squished

**Pull request merged**